### ISSUE TITLE
nvda does not announce the level of lists in browse mode

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#15911]
### Steps to reproduce:
Open a website where there is a list with several levels.
explore the page in browse mode.
example
I opened the nvda user guide page.
I pressed l to be in the first list on the page with 18 items
I navigated with the arrows below until
1.1. General features
### Actual behavior:
list with 6 items
### Expected behavior:
level 2 list with 6 items
In short, I expect to hear the list level when entering and exiting it when navigating with the arrow keys.
This information is spoken when navigating with the right and left arrows character by character.
But I want this same information when browsing line by line.
Logically it can be optional.
I am currently carrying out work that requires me to clearly know the level of each item on the list. I currently need to press the right or left arrows to obtain this information, which delays my work, I do this many times a day.

#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
nvda.exe, NVDA alpha-30299,4519b893

#### Windows version:
windows 10 19045.3693
#### Name and version of other software in use when reproducing the issue:
chrome.exe, Google Chrome 120.0.6099.71

#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes

**Comment by Adriani90** [comment]
@fernando-jose-silva can you reproduce this in firefox as well?

The list level is also announced when using ctrl+left and right arrow keys or when using object navigation.
But it is ignored when using tab or shift+tab or when pressing l or shift+l to navigate from list to list.

I think this request to at least include list level when navigating line by line makes sense. 
I think the list level should not be ignored when entering or leaving the list with tab and shift+tab unless you disable the reporting of lists in document formating settings. People navigating a website in focus mode for example will not know the level of a list.

For convenience it would be nice to hear also the list level when using l and shift+l command. This would increase navigation efficiency in many cases.

The reporting when entering and leaving two nested lists in my opinion should sound like:
"list level one with 18 items
list level two with 6 items;
end of list level two;
end of list level one".

**Comment by fernando-jose-silva** [comment]
in Firefox the results are the same as in Chrome
thanks.

**Comment by jaxcze** [comment]
Hi,
Try my add-on: Last Item Announcer. It's a relatively complex add-on that addresses this issue: https://nvda-addons.janbalak.name/last-item-announcer/en
It is not yet available in the Add-on Store, but it is available on my website.
Best regards,
Jan Balák
